src/manipulation.js

```diff
@@ -306,6 +306,7 @@ jQuery.extend({
 					}
 				}
 				delete data.events;
+				delete elem[dataPriv.expando];
```

Comment:
Missing spaces around the property. Also, can we drop the preceding `delete data.events`?